common/app/common/configuration.scala

```diff
@@ -377,6 +377,10 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
   }
 
   object faciatool {
+    lazy val crossAccountSourceBucket = configuration.getMandatoryStringProperty("frontpress.buckets.source")
+    lazy val sameAccountSourceBucket = configuration.getMandatoryStringProperty("aws.bucket")
```

Comment:
I find the naming here a bit weird.
`sameAccountSourceBucket` ??

And a property called `frontpress.buckets.source`? Maybe we should refer to account names in either properties or values or both.